packages/pages/src/components/Image/Image.tsx

```diff
@@ -0,0 +1,188 @@
+import React from "react";
+import {
+  transformBpSizes,
+  getSrcSetFromSizes,
+  getDefaultSrc,
+  sizesArrayToString,
+  addWidthToCustomSizes,
+} from "./methods";
+import { StandardProps } from "../types";
+import { AspectRatio } from "../AspectRatio";
+
+export interface ImageType {
+  height: number;
+  width: number;
+  url: string;
+}
+
+export interface ImagePropsShared
+  extends React.DetailedHTMLProps<
+      React.ImgHTMLAttributes<HTMLImageElement>,
+      HTMLImageElement
+    >,
+    StandardProps {
+  bpSizes?: [string, string, string];
+  customSizes?: Array<[string, string]>;
+  layout?: "fixed" | "constrained" | "full-width";
+  aspectRatio?: number;
+  objectFit?: React.CSSProperties["objectFit"];
+  objectPosition?: React.CSSProperties["objectPosition"];
+  maxWidth?: number;
+  width?: number;
+  height?: number;
+  /**
+   Options: blurred, dominantColor, or none
+   */
+  placeholder?: string;
+}
+
+export interface ImagePropsWithSrc extends ImagePropsShared {
+  src: string;
+  imageField?: never;
+}
+
+export interface ImagePropsWithField extends ImagePropsShared {
+  src?: never;
+  imageField: ImageType;
+}
+
+export type ImageProps = ImagePropsWithSrc | ImagePropsWithField;
+
+/**
```

Comment:
The JS doc is okay but we should be moving that to the to ts types as it's a more consumer-friendly